crates/weaver_resolver/src/body.rs

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Functions to resolve a semantic convention body.
+
+use weaver_resolved_schema::{
+    body::{Body, BodyField},
+    error::{handle_errors, Error},
+};
+use weaver_semconv::body::{BodyFieldSpec, BodySpec};
+
+/// Resolve a `Body` specification into a resolved `Body`.
+pub fn resolve_body_spec(body: &BodySpec) -> Result<Option<Body>, Error> {
+    match body {
+        BodySpec::Fields { fields } => {
```

Comment:
If fields aren't being resolved, do we need to resolve the whole spec as well?

Reply:
I've kept this function as it contains the `Value` definition, and it keeps the location for any potential resolution of any possible type (maybe) separate (for now) -- may be a candidate for removal based on on-going discussions and just moving this inline.